package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "5.4.2",
+  "version": "5.4.4",
```

Comment:
This is actually considered a [breaking change](https://github.com/LaunchPadLab/lp-components/pull/349#issuecomment-501027020), not a bug fix. The issue is included in Milestone v6, so please set your base branch to `v6`